dwave/system/composites/virtual_graph.py

```diff
@@ -13,6 +13,11 @@
 #    limitations under the License.
 
 """
+Deprecated. 
+Virtual graphs are deprecated due to improved calibration of newer QPUs; to 
+calibrate chains for residual biases, follow the instructions in the 
+`shimming tutorial <https://github.com/dwavesystems/shimming-tutorial>`_.
+
```

Comment:
Consider using [`.. deprecated:: version`](https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-deprecated) Sphinx directive.

Reply:
I did that now for the class (line 50 below).